ports/espressif/boards/m5stack_atom_matrix/sdkconfig

```diff
@@ -0,0 +1,27 @@
+CONFIG_ESP32_ECO3_CACHE_LOCK_FIX=y
```

Comment:
This is now in `esp-idf-config/sdkconfig-esp32.default`, so it doesn't need to be in individual board `sdkconfig`s.
```suggestion
```